nixos/modules/profiles/base.nix

```
@@ -20,7 +20,13 @@
     pkgs.mkpasswd # for generating password files
 
     # Some text editors.
-    pkgs.vim
+    (pkgs.vim_configurable.customize {
```

```suggestion
    (pkgs.vim.customize {
```

The name `vim_configurable` is absolutely [terrible](https://github.com/NixOS/nixpkgs/pull/145351#issuecomment-1072758490) because it suggests you can't customise the basic `vim`, but you [can](https://github.com/NixOS/nixpkgs/pull/164994/commits/21e45db6f1b196678e967b516a0761922bdf5dd7).

This keeps the ISO size at about 828 MiB on master.